src/Modules/UngradedModules/Presentation.jl

```diff
@@ -20,6 +20,7 @@ function presentation(SQ::SubquoModule;
   end
 
   # Old code left for debugging
```

Comment:
I really don't like this kind of thing, this commented code will just bitrot.

If it is a useful for debugging, how about turning it into an actual method? (Obviously not addressed at you, @lgoettgens)

In any case, personally I'd prefer if it was removed here and instead of commented out.